kmod/pfckmod.c

```diff
@@ -56,6 +58,16 @@ static ssize_t pfcCntWr(struct file*          f,
                         char*                 buf,
                         loff_t                off,
                         size_t                len);
+static ssize_t pfcVerboseRd(struct kobject* kobj,
```

Comment:
These two methods, along with the PRINTV above, allow you to enable a "verbose mode" in the kernel driver (by writing `1` to `/sys/modules/pfc/verbose`, which will spit to the kernel log a bit more info useful for diagnosis (I haven't actually used it much).